commit 12f2254b815056bb897a25a0c6dee0122aef52ac
Author: Siddhesh Poyarekar <siddhesh@redhat.com>
Date:   Thu Jul 10 14:19:21 2014 +0530

    Fix -Wundef warning for HAVE_SYS_PARAM_H
    
    Include sys/param.h unconditionally

diff --git a/io/ftw.c b/io/ftw.c
index 7aff7d04c9ed1e95..ffdb4b02dc7ca4d5 100644
--- a/io/ftw.c
+++ b/io/ftw.c
@@ -66,9 +66,7 @@ char *alloca ();
 #include <string.h>
 #include <unistd.h>
 #include <not-cancel.h>
-#if HAVE_SYS_PARAM_H || defined _LIBC
-# include <sys/param.h>
-#endif
+#include <sys/param.h>
 #ifdef _LIBC
 # include <include/sys/stat.h>
 #else
